env.j2

```diff
@@ -1,5 +1,5 @@
 # Flask server configuration.
-FLASK_APP=todo_app/app
+FLASK_APP=todo_app/mongo_app
```

Comment:
I think you undid this file name change, so potentially worth switching this back (although we aren't really maintaining the ansible files anyway, so no real harm either way)

Reply:
name update from mongo_app tp app